README.md

```diff
@@ -45,6 +46,13 @@ $ -DCPM_MaCh3_SOURCE=/path/to/MaCh3/folder
 
 This will overrule the CPMFindPackage command in the CMakeList.txt and will tell CPM to build that instead.
 
+If you have an error about not finding `splineMonolith.h` at the `make install` stage, do: 
+
+~~~~~~~~~~~~~
+$ mv build/_deps/mach3-src/splines/SplineMonolith.h build/_deps/mach3-src/splines/splineMonolith.h
```

Comment:
I think it might be better to have `sed -i "s|splineMonolith|SplineMonolith|g" build/_deps/mach3-src/splines/CMakeLists.txt  [this bug is also fixed on this tag of core https://github.com/mach3-software/MaCh3/tree/DUNECore2024-patches]

Reply:
Ok, I don't want to mess with other experiments! I edited the fix